src/app/utils/field-validation/validators/sectionValidator.ts

```diff
@@ -0,0 +1,19 @@
+import { left, right } from 'fp-ts/lib/Either'
+
+import { ProcessedSingleAnswerResponse } from 'src/app/modules/submission/submission.types'
+import { ResponseValidator } from 'src/types/field/utils/validation'
+
+type sectionValidatorConstructor = () => ResponseValidator<
+  ProcessedSingleAnswerResponse
+>
+
+/**
+ * A function that returns a validation function for a section field when called.
+ */
+const constructSectionValidator: sectionValidatorConstructor = () => (
+  response,
+) => {
+  return response.answer === '' ? right(response) : left('Answer not allowed')
```

Comment:
is this definitely right? if I'm tracing correctly, the current behaviour is that validation for header fields passes even if `response.answer` is null or undefined or an empty array. are there cases in which it might be null/undefined?

Reply:
yup, that's the tech debt i am trying to address by scoping validation more tightly. section fields need to submit an empty string today.